src/madhuraMJun20/scannerClass/ArmstrongNum.java

```diff
@@ -0,0 +1,26 @@
+/*: WAP to check if a number is Armstrong or not.*/
+package madhuraMJun20.scannerClass;
+
+public class ArmstrongNum {
+	void findArmstrong(int input) {
+		int sumOfCube = 0;int cube = 0;	int num1 = 0;int givenNum = input;
+
+		while (givenNum != 0) {
+			num1 = givenNum % 10;
+			cube = num1 * num1 * num1;
+			sumOfCube += cube;
+			givenNum = givenNum / 10;
+
```

Comment:
Do not create extra variable. Please minimize the code. 
void checkNumberIsArmstrongOrNot(int number) {
		double sum =0; 
		int num= number;
		while(num>0) {
			sum = Math.pow(num%10, 3) + sum; 
			num = num/10;
		}
		if(number==sum)
			System.out.println("The given number "+number+" is a Armastrong number.");
		else
			System.out.println("The given number "+number+" is not a Armastrong number.");
	}

Reply:
Changes made to the code..thanks